Services/Certificate/classes/Portfolio/class.ilObjPersistentCertificateVerificationGUI.php

```diff
@@ -1,30 +1,22 @@
-<?php
+<?php declare(strict_types=1);
+
 /* Copyright (c) 1998-2019 ILIAS open source, Extended GPL, see docs/LICENSE */
 
+use ILIAS\DI\Container;
+
 /**
  * @author  Niels Theen <ntheen@databay.de>
  */
 class ilObjPersistentCertificateVerificationGUI
 {
-    /**
-     * @var
-     */
-    private $dic;
-
-    /**
-     * @var ilPortfolioCertificateFileService
-     */
-    private $fileService;
-
-    /**
-     * @var ilLanguage
-     */
-    private $language;
+    private ?Container $dic;
+    private ?ilPortfolioCertificateFileService $fileService;
+    private ?ilLanguage $language;
 
     public function __construct(
-        \ILIAS\DI\Container $dic = null,
-        ilPortfolioCertificateFileService $fileService = null,
-        ilLanguage $language = null
+        ?Container $dic = null,
+        ?ilPortfolioCertificateFileService $fileService = null,
+        ?ilLanguage $language = null
```

Comment:
These types are not optional. Only the passed constructor arguments are optional.